apps.yml

```
@@ -571,6 +571,16 @@ apps:
     url: https://mozillians.org/oidc/authenticate/
     vanity_url:
     - /mozillians
+- application:
+    AAL: LOW
+    authorized_groups:
+    - everyone
+    authorized_users: []
+    client_id: HC1FeMf3dVCTnAZbQR08quMylQEUcu60
+    display: false
+    name: vouches-mozillians
```

This should be a human readable name like `vouches.mozillians.org` or `Mozillians Vouches` or something
```suggestion
    name: vouches.mozillians.org
```